### ISSUE TITLE
Disallow non-english READMEs in trending repositories

### ISSUE THREAD

**Issue by gngeorgiev** [isaacs/github#696]
We see a lot of Chinese repositories which bring no value to the majority of the github users since we cannot read them, nonetheless I would not consider using such a project, they only take up precious trending repo place.

**Comment by cirosantilli** [comment]
Or maybe add language detection to linguist, and allow filtering by language.

**Comment by Ernest314** [comment]
Exactly, it seems selfish to only allow your own language to be used on a website. I second the idea of having language detection/filtering.
